### DISCUSSION TITLE
I accidentally deleted insert_rog and couldn't start milvus，how can I do?

### DISCUSSION THREAD

**Discussion by WangTH37** [milvus-io/milvus#37287] [Q&A and General discussion]
![屏幕截图 2024-10-30 111434](https://github.com/user-attachments/assets/bceda091-b858-4713-9afd-45c0403fc311)

**Comment by yhmo** [comment]
insert_log is the folder on minio to store the data files of milvus. The data is lost if you delete this folder, not able to restore.

**Reply by WangTH37** [reply]
I only delete milvus now ？

**Reply by WangTH37** [reply]
and install again

**Reply by yhmo** [reply]
If you deployed milvus with docker-compose, follow the below steps to recreate a fresh milvus:
1. docker compose down
2. in the same path of the docker-compose.yaml, force delete the folder ".volumes"
3. docker compose up -d

After the milvus container is created, you will see a new ".volumes" folder.
Don't delete this folder, don't delete any folder in the minio, just let milvus to manage the data by itself.

**Reply by xiaofan-luan** [reply]
> I only delete milvus now ？

You have deleted all data in milvus so right now the only way to recover is to clean the .volumes directory and truncate everything
